pysd/py_backend/vensim/vensim2py.py

```diff
@@ -579,13 +580,13 @@ def parse_general_expression(element, namespace=None, subscript_dict=None, macro
     in_ops_list = [re.escape(x) for x in in_ops.keys()]
     pre_ops_list = [re.escape(x) for x in pre_ops.keys()]
     if macro_list is not None and len(macro_list) > 0:
-        macro_names_list = [x['name'] for x in macro_list]
+        macro_names_list = [re.escape(x['name']) for x in macro_list]
     else:
         macro_names_list = ['\\a']
 
     expression_grammar = r"""
     expr_type = array / expr / empty
-    expr = _ pre_oper? _ (lookup_def / build_call / macro_call / lookup_call / call / parens / number / reference) _ (in_oper _ expr)?
+    expr = _ pre_oper? _ (lookup_def / build_call / macro_call / call / lookup_call / parens / number / reference) _ (in_oper _ expr)?
```

Comment:
Order was changed, first we should handle strict situations like function call, builders call and macro calls and then handle non restricted objects, like `reference` and `lookup_call`